test/FeatureFlagTest.php

```diff
@@ -0,0 +1,648 @@
+    {
+        $prop = [
+            "key" => "key",
+            "value" => 1,
+            "operator" => "gt"
+        ];
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 2,
+        ]));
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 3,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => 0,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => -1,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => "23",
+        ]));
+
+        $prop = [
+            "key" => "key",
+            "value" => 1,
+            "operator" => "lt"
+        ];
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 0,
+        ]));
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => -1,
+        ]));
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => -3,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => 1,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => "1",
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => "3",
+        ]));
+
+        $prop = [
+            "key" => "key",
+            "value" => 1,
+            "operator" => "gte"
+        ];
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 1,
+        ]));
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 2,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => 0,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => -1,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => "3",
+        ]));
+
+        $prop = [
+            "key" => "key",
+            "value" => 43,
+            "operator" => "lte"
+        ];
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 0,
+        ]));
+
+        self::assertTrue(FeatureFlag::matchProperty($prop, [
+            "key" => 43,
+        ]));
+
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => 44,
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => "1",
+        ]));
+
+        self::assertFalse(FeatureFlag::matchProperty($prop, [
+            "key" => "3",
+        ]));
+    }
+
+    public function testFlagPersonProperties()
+    {
+        $this->http_client = new MockedHttpClient(host: "app.posthog.com", flagEndpointResponse: MockedResponses::LOCAL_EVALUATION_REQUEST);
+        $this->client = new Client(
+            PROJECT_API_KEY,
```

Comment:
This looks like a real key, replace with random fake one? And/or init host with a random address, rather than `app.posthog.com` ?

Ensures we never accidentally send tests to prod.